### ISSUE TITLE
Move Java Libraries / Frameworks to the website

### ISSUE THREAD

**Issue by trask** [open-telemetry/opentelemetry.io#3040]
We think the Libraries / Frameworks section https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks would be nice to move to the website.

We point users to this often.

It covers both libraries / frameworks supported by the Java agent, as well as those supported via (non-javaagent) "library instrumentation".

**Comment by cartermp** [comment]
I completely agree. Would this live under the automatic instrumentation section?

**Comment by mateuszrzeszutek** [comment]
> Would this live under the automatic instrumentation section?

Most likely, yes.
Although we might want to redesign the docs structure at some point, before implement https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/8946 -- the spring starter is supposed to be an alternative to the javaagent, so there'll be multiple ways to use an auto-instrumentation package.

**Comment by cartermp** [comment]
Oh snap! One thing we can consider is making spring a dedicated "getting started" path since it's so common for Java people. Maybe "getting started" is a top-level folder node with two articles underneath, similar to JS: https://opentelemetry.io/docs/instrumentation/js/getting-started/

**Comment by theletterf** [comment]
@trask Would you be fine with closing this one and open PRs on a library basis?
